src/OrbitCore/SymbolHelper.cpp

```diff
@@ -232,6 +232,10 @@ ErrorMessageOr<ModuleSymbols> SymbolHelper::LoadSymbolsFromFile(const fs::path&
                                         file_path.string(), elf_file_or_error.error().message()));
   }
 
+  if (!elf_file_or_error.value()->HasSymtab()) {
+    return ErrorMessage("ELF file does not have a .symtab section.");
+  }
```

Comment:
in which case this could just call LoadSymbolsFromSymtab if we don't want to fall-back to .dynsym

Reply:
done